helm/cloud-provider-azure/Chart.yaml

```diff
@@ -13,4 +13,4 @@ maintainers:
 name: cloud-provider-azure
 sources:
 - https://github.com/kubernetes-sigs/cloud-provider-azure
-version: 1.26.1
+version: 1.26.2
```

Comment:
By conventionally we've been coupling the version of the helm chart with the most recent, highest-minor version release at the time of the chart bump. So this would become 1.26.4.

When you make this change btw and re-run the update shell script, you might need to clear out the previous updates otherwise the script will think you're adding another version (we want this PR to just ship 1.26.4 and not 1.26.2 _and_ 1.26.4).

Reply:
Done. (The versions were out of sync by necessity in #2994 and we didn't update the chart for v1.26.2 or .3, so I wasn't sure.)